cds_ils/importer/providers/cds/rules/base.py

```diff
@@ -128,9 +128,12 @@ def tags(self, key, value):
 def special_serials(self, key, value):
     """Translates serial fields."""
     _migration = self["_migration"]
-    _serials = _migration["serials"]
+    _serials = _migration.get("serials", False)
     for v in force_list(value):
         result_a = mapping(SERIAL, clean_val("a", v, str))
+        if not result_a or _serials is False:
```

Comment:
when assigning default value _serials  as empty list we most likely don't need ` or _serials is False` check